package-lock.json

```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "mattermost-mobile",
-      "version": "2.4.0",
+      "version": "2.5.0",
```

Comment:
Unrelated changes that just arose from an `npm install`, I presume?

Reply:
yes, 100% accurate your assumption